packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -155,9 +162,11 @@ export class DeltaStorageService implements IDeltaStorageService {
 			async (event) => {
 				const restWrapper = await this.getRestWrapper();
 				const url = this.getDeltaStorageUrl();
+				const reason = fetchReason ?? "No reason provided";
 				const response = await restWrapper.get<ISequencedDocumentMessage[]>(url, {
 					from: from - 1,
 					to,
+					reason,
```

Comment:
Personal preference, but in that case I'd use an empty string instead of providing an arbitrary value like "No reason provided". Is there a need for AFR to know that a reason wasn't provided?